samples/subsys/ipc/ipm_mcux/README.rst

```diff
@@ -0,0 +1,43 @@
+.. _ipm-mcux-sample:
+
+Sample mailbox application
+##########################
+
+Overview
+********
+
+The :ref:`lpcxpresso54114` board has two core processors (Cortex-M4F
+and Cortex-M0+). This sample application uses a mailbox to send messages
+from one processor core to the other.
+
+Requirements
+************
+
+- :ref:`lpcxpresso54114` board
+
+Building and Running
+********************
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/ipc/ipm_mcux
+   :board: lpcxpresso54114
```

Comment:
Need to append `_M4` since the board was renamed

Reply:
fixed